### PR TITLE
Delete CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by MuriloDalRi**
We don't get external contributions and this file is confusing, inconsistent, noise.

https://trello.com/c/iYNuFHCm/259-remove-contributingmd-files-from-repos

⚠️ This repo is Continuously Deployed: make sure you [follow the guidance](https://docs.publishing.service.gov.uk/manual/development-pipeline.html#merge-your-own-pull-request) ⚠️

### CODE REVIEW

**Review by ChrisBAshton — APPROVED** [review]

**Comment by ChrisBAshton** [comment]
But don't merge this for a few days - let's let @KludgeKML merge this when he's finished testing Feedback.

**Review by KludgeKML — APPROVED** [review]
LGTM

**Pull request merged**
